iobroker/docker-compose.yml

```
@@ -0,0 +1,18 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: iobroker_server_1
+      APP_PORT: 8097
```

The port where the server is reachable has to be specified here.
```suggestion
      APP_PORT: 8081
```

---

I actually can’t use port 8098 here, it’s already taken by another Umbrel app.